src/core/algorithms/od/fastod/attribute_set.h

```diff
@@ -0,0 +1,193 @@
+#pragma once
+
+#include <bitset>
+#include <cassert>
+#include <climits>
+#include <cstddef>
+#include <functional>
+#include <string>
+
+#include <boost/functional/hash.hpp>
+
+namespace algos::fastod {
+
+class AttributeSet {
+public:
+    using size_type = size_t;
+
+private:
+    static constexpr size_type kBitsNum = 64;
+
+    std::bitset<kBitsNum> bitset_;
+
+    explicit AttributeSet(std::bitset<kBitsNum> bitset) noexcept : bitset_(std::move(bitset)) {}
+
+public:
+    AttributeSet() noexcept = default;
+
+    explicit AttributeSet([[maybe_unused]] size_type attrs) noexcept {
+        assert(attrs < kBitsNum);
+    }
+
+    explicit AttributeSet([[maybe_unused]] size_type attrs, size_type value) noexcept
+        : bitset_(value) {
+        assert(attrs < kBitsNum);
+    }
```

Comment:
Is it true, that we use only last constructor?

Reply:
It is not true. We use second constructor in function `CreateAttributeSet()` (you can see it [here](https://github.com/yakovypg/Desbordante/blob/fastod/src/core/algorithms/od/fastod/model/attribute_set.h#L158)). Default constructor is required due to using this class in collections.